nixos/tests/vscodium.nix

```diff
@@ -34,35 +34,44 @@ let
       };
       enableOCR = true;
       testScript = ''
+        @failure_mode
+        def codium_running():
+            machine.succeed('pgrep -x codium')
```

Comment:
This reads a bit weird, as if the existence of a codium process is a failure mode.

Reply:
Maybe `@polling_condition` or something, idk.